server/database/config/dbbuild.sql

```diff
@@ -0,0 +1,48 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users, languages, channels, user_channel, user_interest, messages CASCADE;
+
+CREATE TABLE languages (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50) NOT NULL
+);
+CREATE TABLE interests (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL
+);
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  username VARCHAR(255) NOT NULL,
+  email VARCHAR(255) NOT NULL,
+  password VARCHAR(255) NOT NULL,
+  isActive BOOLEAN NOT NULL,
+  bio VARCHAR(255),
+  avatar_path VARCHAR(255),
+  native_lang_id INTEGER NOT NULL REFERENCES languages (id),
+  learning_lang_id INTEGER NOT NULL REFERENCES languages (id)
+);
+
+CREATE TABLE channels (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL
+);
+
```

Comment:
I think it would be better if you put the user channel table right after the channel table